### PR TITLE
feat: Bump org.openapitools to 6.4.0

### PR DESCRIPTION
**Pull request by chberger**

### CODE REVIEW

**Comment by chberger** [comment]
hmm, works on my local Windows machine :-)

Seems like the message is null? Otherwise we wouldn't pass the assertion in 55-56?

An now since @hbelmiro merged #262 we also have a merge conflict in AWXTest

**Comment by hbelmiro** [comment]
@chberger can you please target the `main` branch on your next PRs?

File: deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/template/QuteTemplatingEngineAdapterTest.java [hunk 1]
```diff
@@ -28,13 +30,32 @@ void checkTemplateGenerator() throws IOException {
...
         if (writtenFile != null) {
             writtenFile.deleteOnExit();
             assertEquals("Hello! My name is Jack", new String(Files.readAllBytes(writtenFile.toPath())));
         } else {
             fail("Template failed to write to the file");
         }
     }
+
+    @Test
+    void checkOpenApiSpecIsNotAvailableOnWindows() throws IOException {
```
Comment (ricardozanini) [inline deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/template/QuteTemplatingEngineAdapterTest.java:43]: Is this valid only on Windows? If so, please add `@EnabledOnOs({OS.WINDOWS})`.

**Comment by chberger** [comment]
Removing @hbelmiro wasn't by intention. Just re-requested review from @ricardozanini

**Review by ricardozanini — APPROVED** [review]

**Review by hbelmiro — APPROVED** [review]

**Comment by hbelmiro** [comment]
Cherry-picked to the `main` branch: https://github.com/quarkiverse/quarkus-openapi-generator/pull/278

**Pull request merged**